shell.nix

```diff
@@ -1,8 +1,8 @@
 { inCI ? false }:
 
 with import (fetchTarball {
-  url = https://github.com/NixOS/nixpkgs/archive/be346a1f4bd9bf272c1388b7791cdb0f28bfa2fb.tar.gz;
-  sha256 = "1f0p8x4h5k190vizlan5yljqvsay2krn93jl3m4yqlg808yglsr3";
+  url = https://github.com/NixOS/nixpkgs/archive/6c90920c939ce3e1e7f4c34a7e71759cd3a7765f.tar.gz;
+  sha256 = "1h3z0b8zhpbzv8fjxpa00wy62rar1brszh3yrslm9k2b4bcw1fzj";
```

Comment:
will this require an rebuild of the docker container?

Reply:
Yes. In the end we're using a newer version of nixpkgs with a lot of updated stuff.